### DISCUSSION TITLE
How to make the text file type can share the markdown snippet too?

### DISCUSSION THREAD

**Discussion by OriLiMu** [L3MON4D3/LuaSnip#1055] [Q&A] [answered]
I use lazyvim nvim distro. Which contains friendly snippets and luasnip. I check the doc of luasnip which is too many things, and I havent find anything related to my topic. So How to fix my issue ?

**Comment by L3MON4D3** [comment, marked as answer]
I think there are plenty of issues about stuff like this :wink:
Try `ls.filetype_extend("text", {"markdown"})`, that should make `markdown`-snippets available in buffers with the `text`-filetype.

**Comment by OriLiMu** [comment]
> I think there are plenty of issues about stuff like this 😉 Try `ls.filetype_extend("text", {"markdown"})`, that should make `markdown`-snippets available in buffers with the `text`-filetype.

Thank you. 
I search on the google for "luasnip add snippets file for filetype". Not get much what I want. Did I use the wrong key words to search  ?  I check first 5 or 6 link with this search result but not find related. 

![image](https://github.com/L3MON4D3/LuaSnip/assets/11295355/cdcb3f5a-ca24-4dc9-ae97-90bca9a73382)
